src/tests/ftest/util/general_utils.py

```diff
@@ -651,6 +652,10 @@ def stop_processes(hosts, pattern, verbose=True, timeout=60, added_filter=None):
         verbose (bool, optional): display command output. Defaults to True.
         timeout (int, optional): command timeout in seconds. Defaults to 60
             seconds.
+        added_filter (,optional): negative filter to better identify processes.
+        send_sigusr2 (bool, optional): whether SIGUSR2 should be sent before
```

Comment:
Could this be more generic, like:
```suggestion
        dump_ult_stacks (bool, optional): whether SIGUSR2 should be sent before
```